src/gripper_service/src/gripper_service.cpp

```diff
@@ -0,0 +1,107 @@
+/*Copyright 2023 Brookhaven National Laboratory
+BSD 3 Clause License. See LICENSE.txt for details.*/
+#include <gripper_service/gripper_service.hpp>
+
+using namespace std::placeholders;
+
+GripperService::GripperService()
```

Comment:
This should inherit from Node, not possess one as an attribute. We had to do that in the beam time server for specific functionality, but that is not best practice.

Reply:
Class GripperService now inherits a Node.
https://github.com/maffettone/erobs/blob/9f0d358470cda04c943f454cd38ed3fd825efaa5/src/gripper_service/include/gripper_service/gripper_service.hpp#L17